homeassistant/components/whirlpool/climate.py

```diff
@@ -71,9 +71,7 @@ async def async_setup_entry(
 ) -> None:
     """Set up entry."""
     whirlpool_data: WhirlpoolData = hass.data[DOMAIN][config_entry.entry_id]
-    if not (aircons := whirlpool_data.appliances_manager.aircons):
-        _LOGGER.debug("No aircons found")
-        return
+    aircons = whirlpool_data.appliances_manager.aircons
```

Comment:
Just inline `whirlpool_data.appliances_manager.aircons` below in the for loop? While the existing code already does this, it is a bit odd to reuse the same variable for different types of objects.

Reply:
That's great observation - it really looks like a "mistake" that works but really isn't "good".